pkg/kubectl/cmd/label.go

```diff
@@ -25,12 +25,14 @@ import (
 	cmdutil "github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd/util"
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/resource"
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/runtime"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/util"
 	"github.com/spf13/cobra"
 )
 
 const (
 	label_long = `Update the labels on a resource.
 
+A valid label value is consisted of letters and/or numbers with a max length of %[1]d characters.
```

Comment:
How about mentioning those characters as well?
```
A valid label value is consisted of letters, numbers, and some special characters (hyphen, dot, underscore) with a max length of 63 characters.
```

Imho it feels much more user-friendly than eg.
```
A valid label value has to comply with the ([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9] regural expression with a max length of 63 characters.
```